yast2_firewall-desktop-runner-border-20170209.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "xpos": 336,
+      "ypos": 351,
+      "width": 285,
+      "height": 26,
+      "type": "match"
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "root-auth-dialog"

```

Comment:
well, the needle desktop-runner-border does not exist. I can rename yast2_firewall-desktop-runner-border into desktop-runner-border, this is then better...